extensions/msal-node-extensions/package.json

```diff
@@ -21,7 +21,9 @@
   },
   "scripts": {
     "start": "tsdx watch",
-    "build": "tsdx build",
+    "build": "npm run build:common && npm run link:common && tsdx build",
+    "build:common": "cd ../../lib/msal-common && npm i && npm run build && npm link",
+    "link:common": "npm link @azure/msal-common",
```

Comment:
Doesn't lerna do this? Or is this the `lerna` issue you were referring to?

Reply:
can't use lerna with extensions since it downloads python right? might be good to revisit this in the future though.

Reply:
Makes sense. Thanks.